azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/ConfigMojo.java

```diff
@@ -113,6 +147,9 @@ protected void config(WebAppConfiguration configuration) throws MojoFailureExcep
     protected boolean confirmConfiguration(WebAppConfiguration configuration) throws AzureExecutionException,
         MojoFailureException {
         System.out.println("Please confirm webapp properties");
+        if (StringUtils.isNotBlank(configuration.getSubscriptionId())) {
+            System.out.println("Subscription Id : " + configuration.getSubscriptionId());
```

Comment:
logger.info

Reply:
cannot since logger.info can be disabled by user using higher log levels, this is a confirmation